Source/Core/Common/FileSearch.cpp

```diff
@@ -7,10 +7,19 @@
 
 #include "Common/CommonPaths.h"
 #include "Common/FileSearch.h"
+
+#ifdef _MSC_VER
+#include <experimental/filesystem>
+namespace fs = std::experimental::filesystem;
```

Comment:
Are experimental headers/namespaces guaranteed to keep working in future versions of VS?